manifests/canonical.pp

```diff
@@ -35,11 +35,14 @@
 #
 define postfix::canonical (
   $destination,
-  $file='/etc/postfix/canonical',
+  $file=undef,
   $ensure='present'
 ) {
+  include postfix
   include ::postfix::augeas
 
+  $_file = $file.lest || { "${postfix::confdir}/canonical" }
```

Comment:
I'm guessing this is to avoid defaulting to a value using a potentially uninitialized variable, correct?

Reply:
Commit ammnded.